### PR TITLE
Fix redirect_to :back after upgrading Rails 5

### LINKED ISSUE #347: undefined method `back_url' for #<DialpeerImportsController:0x005612545c17e8>
```
Aug 21 22:41:09 yeti YETI-admin[6461]: [22:41:09.223484 ] [ INFO]: Started GET "/dialpeer_imports/batch_replace" for 1.2.3.4 at 2018-08-21 22:41:09 +0300
Aug 21 22:41:09 yeti YETI-admin[6461]: [22:41:09.225170 ] [ INFO]: Processing by DialpeerImportsController#batch_replace as HTML
Aug 21 22:41:09 yeti YETI-admin[6461]: [22:41:09.345529 ] [ INFO]: Redirected to
Aug 21 22:41:09 yeti YETI-admin[6461]: [22:41:09.349721 ] [ERROR]: undefined method `back_url' for #<DialpeerImportsController:0x005612545c17e8>
Aug 21 22:41:09 yeti YETI-admin[6461]: [22:41:09.349884 ] [ WARN]: /home/yeti-web/vendor/bundle/ruby/2.3.0/gems/actionpack-5.1.6/lib/action_dispatch/routing/polymorphic_routes.rb:249:in `handle_string_call'#012/home/yeti-web/vendor/bundle/ruby/2.3.0/gems/actionpack-5.1.6/lib/action_dispatch/routing/url_for.rb:181:in `full_url_for'#012/home/yeti-web/vendor/bundle/ruby/2.3.0/gems/actionpack-5.1.6/lib/action_dispatch/routing/url_for.rb:167:in `url_for'#012/home/yeti-web/vendor/bundle/ruby/2.3.0/gems/actionpack-5.1.6/lib/action_controller/metal/redirecting.rb:105:in `_compute_redirect_to_location'#012/home/yeti-web/vendor/bundle/ruby/2.3.0/gems/actionpack-5.1.6/lib/action_controller/metal/redirecting.rb:61:in `redirect_to'#012/home/yeti-web/vendor/bundle/ruby/2.3.0/gems/actionpack-5.1.6/lib/action_controller/metal/flash.rb:57:in `redirect_to'#012/home/yeti-web/vendor/bundle/ruby/2.3.0/gems/actionpack-5.1.6/lib/action_controller/metal/instrumentation.rb:64:in `block in redirect_to'#012/home/yeti-web/vendor/bundle/ruby/2.3.0/gems/activesupport-5.1.6/lib/active_support/notifications.rb:166:in `block in instrument'#012/home/yeti-web/vendor/bundle/ruby/2.3.0/gems/activesupport-5.1.6/lib/active_support/notifications/instrumenter.rb:21:in `instrument'#012/home/yeti-
```

### PR DESCRIPTION
**Pull request by workgena**
[Closes #347]

`redirect_to :back` replaced with `redirect_back` in rails 5

### CODE REVIEW

**Comment by senid231** [comment]
@workgena @Fivell I think we can add sugar method to `config/initializers/active_admin.rb` like:
```ruby
def safe_redirect_back(fallback_location = root_path)
  redirect_back(fallback_location: fallback_location)
end
```

**Comment by gigorok** [comment]
@senid231 I think that `redirect_back(fallback_location: fallback_location)` is clear enough & no needs to introduce the new method

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@ def initialize(r, message)
 
   def redirect_to_back(default = root_url)
     if !request.env["HTTP_REFERER"].blank? and request.env["HTTP_REFERER"] != request.env["REQUEST_URI"]
-      redirect_to :back
+      redirect_back fallback_location: root_path
     else
       redirect_to default
     end
```
Comment (gigorok) [inline app/controllers/application_controller.rb]: this method actually do the same thing as `redirect_back fallback_location: root_path` do (except default path of course). I suggest to search by `redirect_to_back` method & replace to a standard rails 5 method `redirect_back ...`

Comment (senid231) [reply app/controllers/application_controller.rb]: method that I've suggested is more DRY, and it's name fully describes it's function

**Pull request merged**